src/licensedcode/detection.py

```diff
@@ -1071,7 +1072,7 @@ def is_correct_detection(license_matches):
     ]
 
     return (
-        all(matcher in ("1-hash", "1-spdx-id", "2-aho") for matcher in matchers)
```

Comment:
> These failing test cases happening when i change this all(matcher in ("1-hash", "1-spdx-id", "2-aho") for matcher in matchers) to all(matcher in ("1-hash", "1-spdx-id") for matcher in matchers) in is_correct_detection.

Oops, my bad, this is actually effecting other parts and not correct. I thought you made the change to add `2-aho` here, but that was not correct, this was present from much earlier. Please revert back to your earlier changes and if tests are all green then this is ready to merge.